### DISCUSSION TITLE
Consultation about using MyST in python / django project

### DISCUSSION THREAD

**Discussion by simkimsia** [executablebooks/meta#616] [General]
Hi team, i like the project of executablebooks in particular MyST. 

I want to see if there's a way i can use it in my Django project. As a way to better learn myst in a real life practical scenario.

I'm willing to pay for consultation like hourly rate, so i can level up on MyST much faster.

Is there someone who can do paid consulting for this?

**Comment by choldgraf** [comment]
Hey @simkimsia - it might help others if you gave an idea of what kinds of things you're trying to learn about. Is there a particular question you have?

MyST is relatively young, so there isn't a huge community of user/developers out there, but if anybody out there wants to help out @simkimsia it would be great :-)

**Reply by chrisjsewell** [reply]
Heya @simkimsia, as @choldgraf said, would be good to get an idea of what you need.
I've had someone ask similar, and I'm never quite sure: I like money lol, but (a) I'm not sure contractually I can accept it 🤷, plus I have barely the time to do the things I'm already doing 😅
But happy to at least have some brief initial discussions

**Reply by simkimsia** [reply]
Hey @chrisjsewell  and @choldgraf  sorry for late reply i got dragged by work, pandemic stuff, family,e tc and wasn't able to respond to this until now.

Basically it's more like I ask questions, no coding required. Ad hoc booking of consultation hr. Maybe 1-2 hrs a month for about 2-3 months is what i foresee. Can do over zoom, etc. And the questions need not be limited to MyST but also cover other executablebooks project.

For instance, right now my top question would be i know that

jupyter-book relies on executablebooks/sphinx-book-theme as default theme, which in turn extends pydata/pydata-sphinx-theme, is there a way for me to come up with a similar theme like executablebooks/sphinx-book-theme but i want a way to add custom meta tags in the html head section. How should i do that?

**Reply by choldgraf** [reply]
one option would be to install + activate the sphinext-opengraph extension, which would let you add arbitrary meta tags to the page: https://github.com/wpilibsuite/sphinxext-opengraph
